bin/ream/src/main.rs

```diff
@@ -166,3 +168,23 @@ pub async fn run_validator_node(config: ValidatorNodeConfig, async_executor: Rea
 
     validator_service.start().await;
 }
+
+pub async fn run_account_manager(mut config: AccountManagerConfig) {
+    info!("starting up account manager...");
+
+    // Validate the configuration
+    config
+        .validate()
+        .expect("Invalid account manager configuration");
+
+    info!(
+        "Account manager configuration: lifetime={}, chunk_size={}",
+        config.lifetime, config.chunk_size
+    );
+
+    let seed_phrase = config.get_seed_phrase();
+    ream_account_manager::generate_keys(&seed_phrase);
+
+    info!("Account manager completed successfully");
+    process::exit(0);
```

Comment:
```suggestion
```

process exit shouldn't be needed right?

Reply:
Without it, it dies with error saying "runtime is dropped within..." It's same with after validator node finishes. It's coming from L42 and L44. One of solutions is declaring executor only when it's used. (e.g., bring those lines into L47)

Reply:
could you move this outside the function and put it at the end of main() then

Reply:
https://github.com/ReamLabs/ream/pull/544/#discussion_r2148809154

As explained above, I'll be adding additional parameters to this command, so if its ok, I'll retain the function as-is for now. @KolbyML

Reply:
Given that it doesn't exit properly with the validator command as well, I think this should be addresses. But it looks like out of this PR's scope.

Reply:
Putting process::exit(0); at the end of main solves it but can we have more elegant solution?

Reply:
I've amended the code as suggested. However, we might need to look into this under a separate issue?  wdyt @jihoonsong @KolbyML